### DISCUSSION TITLE
Addon MD for FP: cannot create the file.md

### DISCUSSION THREAD

**Discussion by Illegit** [EdoFro/Freeplane_MarkdownHelper#23] [Q&A] [answered]
I need help because in the addon last version 0.5 I'm not able to create md file pushing the button note to file.
![image](https://github.com/EdoFro/Freeplane_MarkdownHelper/assets/62674157/51c3c84c-ad44-43b3-9167-a12b3b884130)
Is it a bug or what I'm missing?
FP 1.11.6 and FP 1.11.7_02 with w11.
Thankyou
illegit

**Comment by Illegit** [comment]
Just for your info: I used your old script Note to File (2020-08 about) to create the temp.md file., but with MDHelper nothing happens.

**Comment by EdoFro** [comment, marked as answer]
[Corrected](https://github.com/EdoFro/Freeplane_MarkdownHelper/releases/tag/v0.1.6_pre).

I had also a problem with the Flatlafs Look&Feels when opening the FileChooser dialog to save the file. Now it should work properly.

please report back

**Comment by Illegit** [comment]
Confirmed, thank you, working as expected.
